.changeset/curly-coins-tap.md

```diff
@@ -0,0 +1,52 @@
+---
+"@salt-ds/embla-carousel": major
+---
+
+Carousel is now a stable API.
+
+Features include:
+
+- Composable Salt controls for navigation and progress.
+- Composable with custom content within slides.
+- Built on the popular `embla-carousel` library, with Salt-specific accessibility and theming enhancements.
+
+```
```

Comment:
Although not typical should we just link to the docs page instead of a code block? 

If we want to keep it this example is slightly incorrect, it should probably use an aria-labelledby for the Carousel labelling

Reply:
simplified, we can then update example independent of changelog.